<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QTabWidget" name="tabWidget">
      <property name="contextMenuPolicy">
       <enum>Qt::NoContextMenu</enum>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="movable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="branchTab">
       <attribute name="title">
        <string>Branch</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="3" column="1">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>150</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="addBranchButton">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" rowspan="4">
         <widget class="QListWidget" name="branchList">
          <item>
           <property name="text">
            <string>新建项目</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>新建项目</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>新建项目</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>新建项目</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>新建项目</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>新建项目</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>新建项目</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>新建项目</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>新建项目</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>新建项目</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>新建项目</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>新建项目</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>新建项目</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>新建项目</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>新建项目</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>新建项目</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="deleteBranchButton">
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="checkoutBranchButton">
          <property name="text">
           <string>Checkout</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="commitTab">
       <attribute name="title">
        <string>Commit</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="0" colspan="5">
         <widget class="QListWidget" name="commitList">
          <item>
           <property name="text">
            <string>新建项目</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>新建项目</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>新建项目</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>新建项目</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>新建项目</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>新建项目</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>新建项目</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>新建项目</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>新建项目</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>新建项目</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>新建项目</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>新建项目</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>新建项目</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>新建项目</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QRadioButton" name="orderByDateCommitButton">
          <property name="text">
           <string>Date</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLineEdit" name="searchCommitLineEdit">
          <property name="dragEnabled">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string>Filter in commit messages by input</string>
          </property>
          <property name="clearButtonEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QRadioButton" name="orderByUsernameCommitButton">
          <property name="text">
           <string>Username</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Order by</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="configTab">
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
       <attribute name="title">
        <string>Config</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="1">
         <widget class="QPushButton" name="addConfigButton">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="deleteConfigButton">
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="saveConfigButton">
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>96</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="4" column="1">
         <widget class="QPushButton" name="quickSetButton">
          <property name="text">
           <string>Set name/email</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" rowspan="5">
         <widget class="QTableWidget" name="configTable">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="dragDropOverwriteMode">
           <bool>false</bool>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="showGrid">
           <bool>true</bool>
          </property>
          <property name="gridStyle">
           <enum>Qt::SolidLine</enum>
          </property>
          <property name="sortingEnabled">
           <bool>false</bool>
          </property>
          <property name="rowCount">
           <number>0</number>
          </property>
          <attribute name="horizontalHeaderCascadingSectionResizes">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>300</number>
          </attribute>
          <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderStretchLastSection">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Key</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Value</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>516</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open repo folder</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
